pkg/subscriptions/subscriptions.html

```diff
@@ -0,0 +1,157 @@
+<html>
+<head>
+  <title>Subscriptions</title>
+  <meta charset="utf-8">
+  <link href="../base/cockpit.css" type="text/css" rel="stylesheet">
+  <script src="../base/cockpit.js"></script>
+</head>
+<body>
+    <!-- different containers, depending on current subscription status: updating, unregistered, registered -->
+    <div id="subscriptions-register-failure" class="container-fluid" hidden>
+      <div class="alert alert-danger" role="alert">
+        <span class="pficon-layered > pficon pficon-error-octagon + pficon pficon-error-exclamation"></span>
+          <span><b>System couldn't be registered.</b></span>
+          <span id="subscriptions-register-failure-reason"></span>
+        <button type="button" class="close" aria-label="Close"
+                onclick="$('#subscriptions-register-failure').hide();">
```

Comment:
Bootstrap alerts can be closed without onclick handlers: http://getbootstrap.com/components/#alerts-dismissible